fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -1550,7 +1559,7 @@ struct TenantManagementWorkload : TestWorkload {
 					ASSERT(hasInvalidOption || specialKeysUseInvalidTuple);
 					return Void();
 				} else if (e.code() == error_code_invalid_tenant_configuration) {
-					ASSERT(hasInvalidOption);
+					ASSERT(hasInvalidOption || configuration["assigned_cluster"_sr] == "newcluster"_sr);
```

Comment:
We should also assert above (around L1534) that we aren't changing the cluster name. You could store a boolean at the start to track whether you think you are changing the cluster and then use that in both asserts to follow the convention of the other checks